### ISSUE TITLE
Checking ammo while crouched glitches movement

### ISSUE THREAD

**Issue by Re-VelatoR** [FeeeeK/LCAmmoCheck#2]
Checking ammo or reloading the shotgun while crouched causes the player to move as though they were crouched, even when not crouched. Speed remains as though you were not crouched, and you can see a large portion of your body while moving with items in hand.

**Comment by axd1x8a** [comment]
Yes, I'm aware of this bug but I have no idea why it happens, it's only a visual bug and playing any body animation (like dancing) fixes it

**Comment by Re-VelatoR** [comment]
I'd like to point out this issue isn't present on 1.0.0, so it must be related to the changes made since.

**Comment by axd1x8a** [comment]
> I'd like to point out this issue isn't present on 1.0.0, so it must be related to the changes made since.

Yes, because there is no actual animations and sounds in 1.0.0, it worked by slowing down the reload animation, and caused all animations to slow down. I think that bug was more critical than just a shift in the position of the hands

**Comment by Superluckysu** [comment]
During the use of shotguns, there may still be situations where the "E" is used to pick up items, and when the door is opened, there may be a movement to load ammunition and it is not possible to stand up while loading. It is V1.1.1 that requires squatting or jumping. Anyway, thank you very much for creating the MOD

**Comment by Superluckysu** [comment]
During the use of shotguns, there may still be situations where the "E" is used to pick up items, and when the door is opened, there may be a movement to load ammunition and it is not possible to stand up while loading. It is V1.1.1 that requires squatting or jumping. Anyway, thank you very much for creating the MOD

**Comment by ZorahMagdaro** [comment]
I don't know what mod conflicts with it, I can't see it when I checking ammunition
<img width="178" alt="%7B92FBAA2F-9D59-4742-82D2-752BCD193CB2%7D" src="https://github.com/FeeeeK/LCAmmoCheck/assets/90625091/e8155d76-3d4d-4bac-ab8d-ebdfbb579c94">

**Comment by axd1x8a** [comment]
> I don't know what mod conflicts with it, I can't see it when I checking ammunition

@ZorahMagdaro MoreEmotes?

**Comment by ZorahMagdaro** [comment]
yes thx bro

**Comment by axd1x8a** [comment]
> yes thx bro

Yes, as I thought, moreemotes would not be compatible, see https://github.com/FeeeeK/LCAmmoCheck/issues/3#issuecomment-1902188284

**Comment by ZorahMagdaro** [comment]
Okay, but I don't really like using Toomanyemotes, so I just tested BetterEmotes and it works fine
